src/main/resources/api/service_openapi.yaml

```diff
@@ -206,6 +222,26 @@ components:
         id:
           type: string
           description: UUID of a newly-created workspace
+    WorkspaceDescription:
+      type: object
+      properties:
+        id:
+          description: The ID of the workspace
+          type: string
+          format: uuid
+        spendProfile:
+          description: UUID of provided spend profile
+          type: string
+          nullable: true
+          format: uuid
+        policies:
+          description: Policies provided by the containing folder
+          type: array
+          nullable: true
+          items:
+            type: string
+            format: uuid
```

Comment:
does this current code actually return any policies?

Reply:
no not yet. i added it to the model prematurely- happy to remove it to avoid any potential confusion but in my mind i just wanted to start wiring things through (as much as possible, at elast)

Reply:
I suggest removing it until it exists - so that any client stubs created against this openapi definition will be correct.

Reply:
fair point! removed in latest commit